From 8f30a49c37cdd7deaaa9aab194e7df65517a73ea Mon Sep 17 00:00:00 2001
From: Dmitry Lavnikevich <dmitry.lavnikevich@edality.by>
Date: Sat, 21 Apr 2018 21:53:33 +0300
Subject: [PATCH 1/2] grapeboard: Increase malloc pool size

This solves ubifs mount error caused by the eraseblock buffer allocation fail.
---
 include/configs/grapeboard.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/grapeboard.h b/include/configs/grapeboard.h
index 41ef0fecd3..b131e78e62 100644
--- a/include/configs/grapeboard.h
+++ b/include/configs/grapeboard.h
@@ -51,7 +51,7 @@
 #define CONFIG_LAYERSCAPE_NS_ACCESS
 
 /* Size of malloc() pool */
-#define CONFIG_SYS_MALLOC_LEN				(0x60000 + 128 * 1024)
+#define CONFIG_SYS_MALLOC_LEN				(0x60000 + 128 * 1024 + 0x100000)
 
 /* QSPI */
 #ifdef CONFIG_QSPI_BOOT
-- 
2.17.0

